packages/admin/admin-color-picker/src/FinalFormColorPicker.tsx

```diff
@@ -3,8 +3,14 @@ import { FieldRenderProps } from "react-final-form";
 
 import { ColorPicker, ColorPickerProps } from "./ColorPicker";
 
+/**
+ * @deprecated Use `<ColorField />` instead of `<Field component={FinalFormColorPicker} />`
+ */
 export type FinalFormColorPickerProps = ColorPickerProps & FieldRenderProps<string, HTMLInputElement | HTMLTextAreaElement>;
 
+/**
+ * @deprecated Use `<ColorField />` instead of `<Field component={FinalFormColorPicker} />`
+ */
 export const FinalFormColorPicker = ({ meta, input, ...restProps }: FinalFormColorPickerProps): React.ReactElement => {
```

Comment:
Internally we are still using these components, so I believe we can't really deprecate them. We should rather deprecate the exports in `src/index.ts`

Reply:
They shouldn't be used in the library either, except for the respective field-component...